### PR TITLE
[WIP] Refactor documentation API Reference for gensim.parsing

### PR DESCRIPTION
**Pull request by CLearERR**

### CODE REVIEW

File: gensim/models/keyedvectors.py [hunk 1]
```diff
@@ -278,9 +278,12 @@ def word_vec(self, word, use_norm=False):
         """
         if word in self.vocab:
             if use_norm:
-                return self.syn0norm[self.vocab[word].index]
+                result = self.syn0norm[self.vocab[word].index]
```
Comment (menshikh-iv) [inline gensim/models/keyedvectors.py]: This change isn't related to current PR, please remove this.

File: gensim/parsing/preprocessing.py [hunk 2]
```diff
@@ -63,6 +66,26 @@ def strip_punctuation(s):
 
 
 def strip_tags(s):
+    """Takes string and removes tags
```
Comment (menshikh-iv) [inline gensim/parsing/preprocessing.py]: + newline

File: gensim/parsing/preprocessing.py [hunk 3]
```diff
@@ -63,6 +66,26 @@ def strip_punctuation(s):
...
+    Parameters
+    ----------
+    s : string
```
Comment (menshikh-iv) [inline gensim/parsing/preprocessing.py]: use types from python `str` + newline (between all sections)

File: gensim/parsing/preprocessing.py [hunk 4]
```diff
@@ -63,6 +66,26 @@ def strip_punctuation(s):
...
+    Returns
+    -------
+    string
+        Unicode string without tags
```
Comment (menshikh-iv) [inline gensim/parsing/preprocessing.py]: Points at the end of sentences.

File: gensim/parsing/preprocessing.py [hunk 5]
```diff
@@ -63,6 +66,26 @@ def strip_punctuation(s):
...
+
+
+    Examples
+    --------
+    >>from gensim.parsing.preprocessing import strip_tags
```
Comment (menshikh-iv) [inline gensim/parsing/preprocessing.py]: `>>from gensim.parsing.preprocessing import strip_tags` -> `>>> from gensim.parsing.preprocessing import strip_tags`, here and anywhere.

File: gensim/parsing/preprocessing.py [hunk 6]
```diff
@@ -63,6 +66,26 @@ def strip_punctuation(s):
...
+
```
Comment (menshikh-iv) [inline gensim/parsing/preprocessing.py]: no need to make an empty line here (same for 84)

File: gensim/test/test_doc2vec.py [hunk 7]
```diff
@@ -386,6 +386,12 @@ def test_delete_temporary_training_data(self):
         self.model_sanity(model, keep_training=False)
         self.assertTrue(hasattr(model, 'syn1neg'))
 
+    def test_word_vec_non_writeable(self):
```
Comment (menshikh-iv) [inline gensim/test/test_doc2vec.py]: this isn't relevant too, please remove.

**Comment by CLearERR** [comment]
Another branch will be used for PR.

**Pull request closed without merging**